### PR TITLE
Minor updates to R lesson 1

### PR DESCRIPTION
**Pull request by emilliman5**
Added a tip about 1-based indexing in R vs. 0-based. Made some minor formatting changes

### CODE REVIEW

**Comment by dhaine** [comment]
Thanks @emilliman5 for the fixes, good catches!

File: 01-supp-read-write-csv.Rmd [hunk 1]
```diff
@@ -17,7 +17,7 @@ source('tools/chunk-options.R')
 The most common way that scientists store data is in Excel spreadsheets. While there are R packages designed to access data from Excel spreadsheets (e.g., gdata, RODBC, XLConnect, xlsx, RExcel), users often find it easier to save their spreadsheets in [comma-separated values](reference.html#comma-separated-values-(csv)) files (CSV) and then use R's built in functionality to read and manipulate the data. In this short lesson, we'll learn how to read data from a .csv and write to a new .csv, and explore the [arguments](reference.html#argument) that allow you read and write the data correctly for your needs.
 
 
-### TRead a .csv and Explore the Arguments
+### Tread a .csv and Explore the Arguments
```
Comment (tomwright01) [inline 01-supp-read-write-csv.Rmd]: Tread?

File: 01-starting-with-data.Rmd [hunk 2]
```diff
@@ -209,6 +209,10 @@ The [slice](reference.html#slice) `1:4` means, "Start at index 1 and go to index
 
 The slice does not need to start at 1, e.g. the line below selects rows 5 through 10:
 
+> ## Tip {.callout} 
+> R indexes are 1-based, not 0-based. That means that first element has an index of 1 instead of 0.
```
Comment (tomwright01) [inline 01-starting-with-data.Rmd]: Since nothing else in R is 0 based is this really necessary?

Comment (jdblischak) [inline 01-starting-with-data.Rmd]: I agree this is confusing. This lesson is targeted at novice programmers. One nice thing about R is that we do not have to spend time discussing 0-indexing. Why would novices expect it to be any other way?

If we do want to make note of it, we should at least give them the context: explain that some other languages, e.g. Python, start indexing from zero.

**Comment by emilliman5** [comment]
My thought was that an experienced Python/Perl/... programmer can still be a novice R programmer. I agree more context should be given as to why this matters.

**Comment by dhaine** [comment]
Sorry, `Tread`should have been `Read`.

**Comment by dhaine** [comment]
I have mixed feelings on the second point. At first I got the same opinion than @emilliman5, i.e. we don't really have totally complete novice participants, They would have (some) knowledge of something else and a single note about 0/1-index without context would be good enough. Now not sure if better to add context or just not talk about it (would prefer the latter). Do you want me to revert the merge?

**Comment by jdblischak** [comment]
While it is true that workshop attendees come in with a variety of backgrounds, this lesson was written explicitly with a novice programmer in mind. This is evident throughout. For example, when introducing functions, it spends a large amount of time motivating the use of functions and explaining the idea of encapsulation. Similarly, the loops section starts with the motivations to use loops. An experienced Python/Perl/... programmer already knows why functions and loops are useful, and this lesson would be an inefficient means for them to pick up R syntax.

If we do keep the tip, it needs to have context. For example, it could say something like "R indexes start counting at 1, which is quite natural. However, be careful when learning to read/write code in other programming languages because some, e.g. Python, start counting at 0."

Since this PR contained multiple changes, it may be easier to just submit another PR to make these additional changes instead of reverting these commits. But I leave that decision to you.

File: 01-supp-read-write-csv.Rmd (hunk 1 above)
Comment (chendaniely) [inline 01-supp-read-write-csv.Rmd]: Probably meant to type `Read`?

**Comment by dhaine** [comment]
Sorry for the delay to reply. I like very much the example given by @jdblischak . @emilliman5 , do you want to submit an other PR for this?

**Pull request merged**